tox.ini

```diff
@@ -21,22 +21,20 @@ covercmd =
       --cover-branches \
       --nocapture
 
-# Until grpcio 0.13.1 ships, this environment is broken on UCS2 builds.
-# See: https://github.com/grpc/grpc/issues/5280 and
-#      https://github.com/grpc/grpc/pull/5319
-#[testenv:py27]
-#basepython =
-#    python2.7
-#deps =
-#    {[testenv]deps}
-#    grpcio >= 0.13.0
-#setenv =
-#    PYTHONPATH =
+[testenv:py27]
+basepython =
+    python2.7
+commands =
+    pip install gcloud[grpc]
```

Comment:
This doesn't feel like the `tox`-way (like Pythonic, though even more vague).

ISTM this will slow down our core test suite since `tox` will check all the deps at the beginning (talking to the PyPI server) and then at test run time will **also** check the deps.